features/supplier/view_and_edit_services.feature

```diff
@@ -0,0 +1,27 @@
+@view_dos_service
+Feature: Supplier being able to view their DOS services
+
+Background:
+  Given I have a live digital outcomes and specialists framework
+  And I have a supplier
+  And that supplier is logged in
+  Given that supplier is on that framework
+
+Scenario Outline: Supplier coming from dashboard to view the detail page for their <lot_slug> service
```

Comment:
Scenario Outline uses Examples to parametrize and as such it's like a collection of Scenarios, it's pretty cool :). Still, I removed lot_slug from the title, as more general title is fine, you're right.